packages/ibm-products/src/components/NotificationsPanel/NotificationsPanel.figma.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright IBM Corp. 2025
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React from 'react';
+import { NotificationsPanel } from './NotificationsPanel';
+import figma from '@figma/code-connect';
+
+const connectionURL =
+  'https://www.figma.com/design/0F9dKH2abAd7gSfvnacfWf?node-id=10574%3A240266';
+
+const sharedProps = {
+  title: figma.nestedProps('Header', {
+    text: figma.textContent('"Notifications"'),
+  }),
+  doNotDisturbLabel: figma.nestedProps('Toggle', {
+    text: figma.textContent('Value'),
+  }),
+};
+
+figma.connect(NotificationsPanel, connectionURL, {
+  variant: { 'Empty state': false },
+  props: {
+    ...sharedProps,
+  },
+  example: (props) => {
+    const notifications = [
+      {
+        id: '1',
+        type: 'success',
+        title: 'Title.',
+        description: 'Message',
+        timestamp: new Date(new Date().getTime() - 30 * 1000),
+        unread: true,
+        onNotificationClick: () => console.log('Clicked on notification'),
+      },
+      {
+        id: '2',
+        type: 'error',
+        title: 'Title',
+        description: 'Message',
+        timestamp: new Date(new Date().getTime() - 11 * 1000),
+        unread: false,
+        onNotificationClick: () => console.log('Clicked on notification'),
+      },
+      {
+        id: '3',
+        type: 'warning',
+        title: 'Title',
+        description: 'Message',
+        timestamp: new Date(new Date().getTime() - 120 * 1000),
+        unread: false,
+        onNotificationClick: () => console.log('Clicked on notification'),
+      },
+    ];
+    return (
```

Comment:
If we put the array loose in the `data` prop, we can remove the `return` and the encapsulating curly braces